src/proxy/http/profiles/mod.rs

```diff
@@ -0,0 +1,286 @@
+extern crate linkerd2_router as rt;
+extern crate tower_discover;
+
+use futures::Stream;
+use http;
+use indexmap::IndexMap;
+// Import is used by WeightedIndex::sample.
+#[allow(unused_imports)]
+use rand::distributions::Distribution;
```

Comment:
I've confirmed that you can actually just delete this import from this file. It is needed in another file though.